### ISSUE TITLE
Back buttons do not work when JavaScript is disabled

### ISSUE THREAD

**Issue by stuartmccoll** [heritagefund/funding-frontend#239]
We're displaying back buttons on a small subset of pages within the service. These buttons are just using browser back functionality to direct a user back to the previous page within their browser history. This doesn't work when a user has JavaScript disabled. We should only show these buttons if a user has JavaScript enabled.
